os/net/lwip/src/netif/ethernet.c

```diff
@@ -112,7 +112,21 @@ err_t ethernet_input(struct pbuf *p, struct netif *netif)
 
 	/* points to packet payload, which starts with an Ethernet header */
 	ethhdr = (struct eth_hdr *)p->payload;
-	LWIP_DEBUGF(ETHARP_DEBUG | LWIP_DBG_TRACE, ("ethernet_input: dest:%" X8_F ":%" X8_F ":%" X8_F ":%" X8_F ":%" X8_F ":%" X8_F ", src:%" X8_F ":%" X8_F ":%" X8_F ":%" X8_F ":%" X8_F ":%" X8_F ", type:%" X16_F "\n", (unsigned)ethhdr->dest.addr[0], (unsigned)ethhdr->dest.addr[1], (unsigned)ethhdr->dest.addr[2], (unsigned)ethhdr->dest.addr[3], (unsigned)ethhdr->dest.addr[4], (unsigned)ethhdr->dest.addr[5], (unsigned)ethhdr->src.addr[0], (unsigned)ethhdr->src.addr[1], (unsigned)ethhdr->src.addr[2], (unsigned)ethhdr->src.addr[3], (unsigned)ethhdr->src.addr[4], (unsigned)ethhdr->src.addr[5], lwip_htons(ethhdr->type)));
+	LWIP_DEBUGF(ETHARP_DEBUG | LWIP_DBG_TRACE,
+				("ethernet_input: dest:%" X8_F ":%" X8_F ":%" X8_F ":%" X8_F ":%" X8_F ":%" X8_F ",src:%" X8_F ":%" X8_F ":%" X8_F ":%" X8_F ":%" X8_F ":%" X8_F ",type:%" X16_F "\n",
```

Comment:
Actually, X8_F evaluates to "02" in `arch.h`. Perhaps it needs a correction to "02x". Please ignore if not needed.

Reply:
I think I don't need to modify it